lib/auth/webauthncli/register.go

```diff
@@ -0,0 +1,279 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webauthncli
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+
+	"github.com/duo-labs/webauthn/protocol"
+	"github.com/duo-labs/webauthn/protocol/webauthncose"
+	"github.com/flynn/u2f/u2ftoken"
+	"github.com/fxamacker/cbor/v2"
+	"github.com/gravitational/teleport/api/client/proto"
+	"github.com/gravitational/trace"
+
+	wanlib "github.com/gravitational/teleport/lib/auth/webauthn"
+	log "github.com/sirupsen/logrus"
```

Comment:
These imports should be separated into chunks in the order: 
- standard 
- gravitational 
- other

Reply:
I have been using the goimports-like style in most (all?) of my PRs, which roughly goes:

* standard
* <newline>
* others

I also like splitting aliased imports in a 3rd block, and side-effect ones in a forth, giving:

* standard
* <newline>
* others
* <newline>
* aliased
* <newline>
* "_" imports

All of those pass the linters (as do the various other forms we see through the project). Since we seem to lack a consistent style and linters are happy, I'll push back here. Happy to talk about this if you want.

Reply:
I've just been told by various other people that this is the way we format imports, so was just passing it along. Although some files lack consistency, we should at least try to keep this format if we remember.

Reply:
Ok, I won't make promises now, but I'll take a note to trace back the files and format accordingly. Much better would be to have this captured in an automated linter/formatter, so we don't have to think about this.